cypress/integration/adapters.ts

```diff
@@ -0,0 +1,61 @@
+import { loadFlowCode } from '../support/helper';
+// tslint:disable: quotemark
+/// <reference types="Cypress" />
+
+
+describe('Adapter install', () => {
+
+  beforeEach(() => {
+    // Prevent external network request for adapter config
+    cy.intercept('GET', 'https://kendraio.github.io/kendraio-adapter/config.json', {
+      fixture: 'adapterConfig.json'
+    }
+    ).as('adapterConfig.json');
+    cy.intercept('GET', 'https://kendraio-adapter.kendraio.now.sh/bloomen.json', {
+      fixture: 'bloomen.json'
+    }
+    ).as('bloomen.json');
+
+  });
+
+  it('Export a packaged adapter with attachments', () => {
+    // first go to settings and allow core actions in flows
+        cy.visit('/core/settings');
+        cy.get('#formly_16_boolean_exposeCoreActions_5 .mat-checkbox-inner-container').click();        
```

Comment:
I wonder how well this selector will hold.

Reply:
Took a while to find a selector that eventually worked. I failed to get it to work with a "contains" statement, as there are a number of nested elements, that seemed to get in the way. I hope that the ID of the field will be fairly static.